includes/admin/downloads/metabox.php

```diff
@@ -138,16 +136,16 @@ function edd_download_meta_box_save( $post_id, $post ) {
 			}
 
 			update_post_meta( $post_id, $field, $new_default_price_id );
+		} elseif ( '_edd_product_type' === $field && '0' === $_POST[ $field ] ) {
```

Comment:
@LisaCee Let's add a comment here explaining that for backwards compatibility no value is stored when Product Type is set to "Default" (represented by a `0` value).